pkg/k8shandler/curation.go

```diff
@@ -48,7 +48,7 @@ func CreateOrUpdateCuration(cluster *logging.ClusterLogging) (err error) {
 
 		printUpdateMessage := true
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
+			if exists := logging.DoesClusterLoggingExist(cluster); exists {
```

Comment:
shouldn't this be `cluster.Exists()` ?

Reply:
This PR only does a partial refactoring to minimize impact across the code base.  it handles changes related to the collector